shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -207,7 +207,7 @@
   settings.enable_wide_gamut = enableWideGamut;
 
   // Whether to enable Impeller.
-  NSNumber* enableImpeller = [mainBundle objectForInfoDictionaryKey:@"FLTEnableImpeller"];
+  NSNumber* enableImpeller = [bundle objectForInfoDictionaryKey:@"FLTEnableImpeller"];
```

Comment:
This needs to check both, right?  Otherwise we will break customers who followed our instructions to add the flag to their app's Info.plist.

Reply:
Something like this:
```suggestion
  NSNumber* enableImpellerMainBundle = [mainBundle objectForInfoDictionaryKey:@"FLTEnableImpeller"];
  NSNumber* enableImpeller =
      enableImpellerMainBundle != nil
          ? enableImpellerMainBundle
          : [bundle objectForInfoDictionaryKey:@"FLTEnableImpeller"];
```

Reply:
Thanks! Based on the comment above, I look in the app bundle first.